designer_v2/lib/localization/app_en.arb

```diff
@@ -537,6 +536,35 @@
   "code_list_empty_description": "Add participants to your study via access codes.",
   "code_list_header_code": "Code",
   "action_button_code_new": "New code",
+  "@__________________STUDYPAGE_MONITOR__________________": {},
+  "participant_details_title": "Participant Details",
+  "participant_details_study_days_overview": "Study days overview",
+  "participant_details_study_days_description": "This section provides an overview of the participant's study days.",
+  "participant_details_color_legend_title": "Color Legend",
+  "participant_details_color_legend_completed": "Completed",
+  "participant_details_color_legend_partially_completed": "Partially Completed",
+  "participant_details_color_legend_missed": "Missed",
+  "monitoring_no_participants_title": "There are no participants in this study yet",
+  "monitoring_no_participants_description": "Once participants have enrolled in your study, you can monitor their progress and view their data here.",
+  "monitoring_participants_title": "Participants",
+  "monitoring_total": "Total",
+  "monitoring_active": "Active",
+  "monitoring_active_tooltip": "Number of participants who are currently in the study",
+  "monitoring_inactive": "Inactive",
+  "monitoring_inactive_tooltip": "Number of participants who are inactive 3 days in a row",
+  "monitoring_dropout": "Dropout",
+  "monitoring_dropout_tooltip": "Number of participants who have dropped out before the end of the study",
+  "monitoring_completed": "Completed",
+  "monitoring_completed_tooltip": "Number of participants who reached the end of the study",
+  "monitoring_table_column_participant_id": "Participant ID",
+  "monitoring_table_column_invite_code": "Invite Code",
+  "monitoring_table_column_enrolled": "Started At",
+  "monitoring_table_column_last_activity": "Last Activity",
+  "monitoring_table_column_day_in_study": "Day in Study",
+  "monitoring_table_column_completed_interventions": "Completed\nInterventions",
+  "monitoring_table_column_completed_surveys": "Completed\nSurveys",
+  "monitoring_table_row_tooltip_dropout": "This participant has dropped out and no new activity will be added",
+  "monitoring_table_completed_interventions_header_tooltip": "Intervention is only completed, if all tasks have been completed",
```

Comment:
Revise the capitalization to use sentence case even in headings following [material UI's best practice](https://m3.material.io/foundations/content-design/style-guide/ux-writing-best-practices#fc5c2a78-f4bf-4d42-bdac-42ff80391129)